core/admin/start.py

```diff
@@ -4,4 +4,12 @@
 
 os.system("flask mailu advertise")
 os.system("flask db upgrade")
+
+if 'INITIAL_ADMIN_ACCOUNT' in os.environ and 'INITIAL_ADMIN_DOMAIN' in os.environ and 'INITIAL_ADMIN_PW'  in os.environ:
```

Comment:
Yes, POSTMASTER @  DOMAIN is needed, but still, first you need an admin account :-)
Anyway, I would not use POSTMASTER for INITIAL_ADMIN_ACCOUNT since I think the common use is to create an alias from postmaster to whatever real account (that's how I use it).

Using DOMAIN - yes, that could be a variant. Maybe I should allow use of INITIAL_ACCOUNT_DOMAIN variable, but if it is not set (but INITIAL_ADMIN_ACCOUNT is), use the value from DOMAIN. Also, in the documentation I already wrote:

> Most probably identical to the DOMAIN variable

Reply:
Ok, I get it. I will close this comment. @kaiyou's comment still needs to be implemented tho.